c++/file2array.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+# Copyright 2017 Amazon.com and it's affiliates
+# Copyright 2015 Cloudera Inc.
+# Copyright 2013 University of Chicago and Argonne National Laboratory
```

Comment:
I am not sure if this Copyright information and the Apache License can co-exist.